herbert_core/classes/MPIFramework/@ClusterWrapper/ClusterWrapper.m

```diff
@@ -294,17 +294,17 @@
                 user_path = curr_path(~startsWith(curr_path,existing_addpath));
             end
             % Build user path
-            user_path = strjoin(user_path, pathsep);
 
             if isempty(existing_addpath)
-                obj.common_env_var_('MATLABPATH') = necessary_addpath;
+                addpath = {necessary_addpath};
```

Comment:
`addpath` is an existing matlab function. Do not shadow it here.

Reply:
finaly useful comment